plugins/modules/kms_key.py

```diff
@@ -154,6 +160,7 @@
 # Create a new KMS key
```

Comment:
I would personally keep this example as it is and add another that explicitly states "Create a multi-region KMS key" or so.